speech/api/speech_async_rest.py

```diff
@@ -57,8 +57,12 @@ def main(speech_file):
     service_request = service.speech().asyncrecognize(
         body={
             'config': {
-                'encoding': 'LINEAR16',
-                'sampleRate': 16000
+                # There are a bunch of config options you can specify. See
+                # https://cloud.google.com/speech/reference/rest/Shared.Types/RecognitionConfig  # noqa
```

Comment:
We are the linter team :)
What's the desired behavior? We still want to catch long lines, and we still want a way to tell the linter to ignore long lines that for one reason or another we wish to allow..